src/styles/variables.css

```diff
@@ -102,6 +130,44 @@
 	--theme-shadow-opacity-inner: 0.06;
 }
 
+.theme-light-blue {
+	--theme-color-primary-rgb: var(--theme-color-blue-rgb);
+	--theme-color-primary-50: var(--theme-color-blue-50);
+	--theme-color-primary-100: var(--theme-color-blue-100);
+	--theme-color-primary-200: var(--theme-color-blue-200);
+	--theme-color-primary-300: var(--theme-color-blue-300);
+	--theme-color-primary-400: var(--theme-color-blue-400);
+	--theme-color-primary-500: var(--theme-color-blue-500);
+	--theme-color-primary-600: var(--theme-color-blue-600);
+	--theme-color-primary-700: var(--theme-color-blue-700);
+	--theme-color-primary-800: var(--theme-color-blue-800);
+	--theme-color-primary-900: var(--theme-color-blue-900);
+
+	--theme-background-color: var(--theme-white);
+	--theme-secondary-background-color: var(--theme-color-secondary-100);
+
+	--theme-text-color: var(--theme-color-secondary-900);
+	--theme-secondary-text-color: var(--theme-color-secondary-700);
+
+	--theme-scrollbar-color: var(--theme-color-primary-100);
+	--theme-scrollbar-hover-color: var(--theme-color-primary-200);
+
+	--theme-shadow-color: 0, 0, 0;
+
+	--theme-shadow-opacity-xs: 0.05;
+	--theme-shadow-opacity-sm: 0.05;
+	--theme-shadow-opacity-base-primary: 0.1;
+	--theme-shadow-opacity-base-secondary: 0.06;
+	--theme-shadow-opacity-md-primary: 0.1;
+	--theme-shadow-opacity-md-secondary: 0.06;
+	--theme-shadow-opacity-lg-primary: 0.1;
+	--theme-shadow-opacity-lg-secondary: 0.05;
+	--theme-shadow-opacity-xl-primary: 0.1;
+	--theme-shadow-opacity-xl-secondary: 0.04;
+	--theme-shadow-opacity-2xl: 0.25;
+	--theme-shadow-opacity-inner: 0.06;
```

Comment:
i think we don't need these, because we already have the same values in `.theme-light`